fhir-persistence/src/main/java/com/ibm/fhir/persistence/context/impl/FHIRSystemHistoryContextImpl.java

```diff
@@ -51,6 +81,14 @@ public void setAfterHistoryId(long id) {
         this.afterHistoryId = id;
     }
 
+    /**
+     * Add the resource type to include in the response
+     * @param resourceType
+     */
+    public void addResourceType(String resourceType) {
+        this.resourceTypes.add(resourceType);
+    }
```

Comment:
does casing matter?

Reply:
Resource types are checked via ModelSupport, so yes if I undertood the question correctly

Reply:
ok, I was debating whether 
A. the javadoc should reflect that; or
B. that should be enforced here;

but I guess we pretty consistently use the TitleCase/CamelCase for resource names throughout the project, so its probably fine